extensions/wikia/Recirculation/services/FandomDataService.class.php

```diff
@@ -79,7 +79,7 @@ private function apiRequest( $type ) {
 	private function buildUrl( $type, $options ) {
 		switch ( $type ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1301)